department-of-veterans-affairs/vets-api
# API-38635-hard-code-claimant-web-service

## Summary

- Adds test for claimantServiceBean
- Adds ClaimantServiceBean/ClaimantWebService to CACHED_SERVICES 
*****NEED TO ADD BACK AFTER MERGE CONFLICT

## Related issue(s)

- [API-38635](https://jira.devops.va.gov/browse/API-38635)

## Testing done

- To test in Postman
1. call `{{uri}}/services/claims/v1/forms/2122`
2. place break point [here](https://github.com/department-of-veterans-affairs/vets-api/blob/API-38535-enable-hardcoded-wsdl-org-web-service/modules/claims_api/lib/bgs_service/local_bgs.rb#L320)
3. look at the endpoint variable, and it should be `ClaimantServiceBean/ClaimantWebService` 
4. determine if the definition is found
5. Does the call work?


## What areas of the site does it impact?
	modified:   modules/claims_api/lib/bgs_service/local_bgs.rb
	modified:   modules/claims_api/spec/lib/claims_api/find_definition_spec.rb

## Acceptance criteria
